apps/judicial-system/web/src/components/AccordionItems/CaseFilesAccordionItem/CaseFilesAccordionItem.tsx

```diff
@@ -129,8 +134,8 @@ const CaseFilesAccordionItem: React.FC<Props> = (props) => {
               onClick={() => uploadFilesToCourt(workingCase.caseFiles)}
               loading={uploadState === UploadState.UPLOADING}
               disabled={
-                uploadState === UploadState.UPLOADING ||
-                uploadState === UploadState.ALL_UPLOADED
+                uploadState !== UploadState.SOME_NOT_UPLOADED &&
+                uploadState !== UploadState.UPLOAD_ERROR
```

Comment:
Disable the upload button if no file can be uploaded (and the button is visible).